utils/RunONNXModel.py

```diff
@@ -1138,22 +1231,55 @@ def run_performance_test(self, output_name=None, input_feed=None, **kwargs):
             return outs
 
 
-################################################################################
-# Standalone driver
+def load_user_config(config_file):
```

Comment:
Would you maybe include in a comment a possible config_file, so that if we want to generate one, we have an example to build from?